### PR TITLE
optimize comment of package-info.java. (#1273)

### LINKED ISSUE #1273: error code-formatter of comment for package-info.java
### Describe the bug

I found that the all comment for package-info.java in this project is error in code-formatter; Should we fix it ?

For example,
In current code, the comment is under the package-name in package-info.java , the tips of comment would not occurred when the pointer is focus on the package.
image

When i move the comment above the package-name in package-info.java, the tips will occur.
image

### Environment

- SOFARPC version:
- JVM version (e.g. `java -version`):
- OS version (e.g. `uname -a`):
- Maven version:
- IDE version:

### PR DESCRIPTION
**Pull request by jnan806**
Signed-off-by: Jiangnan Jia <<jnan0806@gmail.com>>

### Result:

Fixes #1273

### CODE REVIEW

**Review by EvenLjj — APPROVED** [review]
LGTM

**Comment by EvenLjj** [comment]
@jnan806 Please execute `mvn clean compile` to format code before submitting PR.
image

**Comment by jnan806** [comment]
get it and has finished.

File: core/api/src/main/java/com/alipay/sofa/rpc/bootstrap/package-info.java [hunk 1]
```diff
@@ -14,8 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.alipay.sofa.rpc.bootstrap;
-
 /**
```
Comment (EvenLjj) [inline core/api/src/main/java/com/alipay/sofa/rpc/bootstrap/package-info.java:17]: @jnan806 It's better to leave one line blank. The format will look better.

Comment (jnan806) [reply core/api/src/main/java/com/alipay/sofa/rpc/bootstrap/package-info.java:17]: the blank line was deleted by exec `mvn clean package` which was requested by following.

image

Comment (EvenLjj) [reply core/api/src/main/java/com/alipay/sofa/rpc/bootstrap/package-info.java:17]: OK

**Review by EvenLjj — APPROVED** [review]
LGTM

**Review by OrezzerO — APPROVED** [review]
LGTM

**Pull request merged**